x-pack/plugins/threat_intelligence/public/modules/indicators/components/indicators_table/cell_renderer.tsx

```diff
@@ -63,6 +64,9 @@ export const cellRendererFactory = (from: number) => {
       return displayValue(indicator) || EMPTY_VALUE;
     }
 
-    return unwrapValue(indicator, columnId as RawIndicatorFieldId) || EMPTY_VALUE;
+    const fieldType: string = fieldTypesMap[columnId];
```

Comment:
what about abstracting this logic into a component of some sort? I can see that we will have the need to render indicator fields in many different occasions and contexts. I don't know what exactly will be the right abstraction, but so I can see for example a component that accepts `indicator` and the `key` to render and hides this `unwrapValue` and date formatting magic inside. wdyt?

Reply:
yup I can try that as well!
I feel like we're already using a table, cell-renderer factory, a action-row-cell, we would then use another abstracted component... that might be a bit much 🤣 I'll see how it looks!

Reply:
this might have been solved at the same time [here](https://github.com/elastic/kibana/pull/138106#discussion_r940047702)

Reply:
I think the approach works, I'd just implement it as a simple component, left a comment below